edge-modules/MessagesAnalyzer/Controllers/ReportController.cs

```diff
@@ -15,14 +15,14 @@ public ActionResult<string> Get()
         {
             string resultJson = Reporter.GetReceivedMessagesReport(Settings.Current.ToleranceInMilliseconds).ToString();
 
-            if (Settings.Current.LogAnalyticEnabled)
+            if (Settings.Current.LogAnalyticsEnabled)
```

Comment:
@and-rewsmith In which PR, are you going to do the three PostAsync() thingy?

Reply:
I did this to keep consistent with one of venkat's log modules, which refers to it as a secret. That PostAsync() change is in the twin update PR.